### DISCUSSION TITLE
How to check if Data doesn't exist in Condition builder

### DISCUSSION THREAD

**Discussion by sparkls0** [AutomaApp/automa#1340] [Q&A]
Hey there, I can't really grasp my mind around how we can check if a variable doesn't exist 

I tried 'Data exists' !variables@variableName

doesn't seem to work

How can we do that?

Thanks in advance

<img width="765" alt="CleanShot 2023-08-26 at 23 48 29@2x" src="https://github.com/AutomaApp/automa/assets/48011728/caca73da-ed09-44de-af7a-fbd1ae3292b2">

**Comment by sparkls0** [comment]
<img width="1367" alt="CleanShot 2023-08-27 at 00 13 35@2x" src="https://github.com/AutomaApp/automa/assets/48011728/d39350ec-d352-4241-a8a1-98ac78111e41">

well I couldn't check inside one condition if variable exists + variable doesn't exist + variable doesn't exist, but I found a way of checking each one by making one condition at a time, and using fallbacks.

That works, but I feel like there is better to be done in Automa, am I right?

**Reply by SuperTempAccount** [reply]
Hi have you tried adding different paths instead of conditions?
I believe this is what you're looking for 
<img width="763" alt="image" src="https://github.com/AutomaApp/automa/assets/83566267/91a3d927-fee6-43b4-97e0-16f54666e945">

Each path needs only one Data Exists check.
